lib/models/addon.js

```diff
@@ -730,7 +777,7 @@ var Addon = CoreObject.extend({
     if (addonTemplates) {
       standardTemplates = new Funnel(addonTemplates, {
         srcDir: '/',
-        destDir: 'modules/' + this.name + '/templates'
+        destDir: this.name + '/templates'
```

Comment:
@twokul do they depend on this, or the later `modules` funnel? I don't believe the later is removed (yet) although we do intend to explore this soon.